pallets/gear-scheduler/src/lib.rs

```diff
@@ -243,6 +243,11 @@ pub mod pallet {
         fn reservation() -> Self::Cost {
             T::ReservationCost::get()
         }
+
+        fn dispatch_stash() -> Self::Cost {
+            // TODO: maybe we have to change value of gas charging for dispatch stashs
+            T::WaitlistCost::get()
```

Comment:
incorrect value

Reply:
As we discussed before we do the same cost for DispatchStash and Waitlist. What is your suggestions?

Reply:
done